sdk/tests/conformance2/uniforms/gl-uniform-arrays-sub-source.html

```diff
@@ -258,11 +258,19 @@
 
 ];
 
-function checkUniformValues(program, typeInfo) {
+function checkUniformValues(program, typeInfo, initialValues) {
```

Comment:
Could you rename "initialValues" to "useInitialValues"? Right now it sounds like an array rather than a boolean.